package.json

```diff
@@ -22,38 +22,41 @@
     "lib"
   ],
   "main": "./lib/podlet.js",
-  "types": "index.d.ts",
+  "types": "./types/podlet.d.ts",
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
-    "test": "tap --no-check-coverage"
+    "test": "tap --no-check-coverage",
+    "types": "tsc --declaration --emitDeclarationOnly && echo 'declare global {\n  namespace Express {\n    interface Response {\n      podiumSend(fragment: string, ...args: unknown[]): Response;\n    }\n  }\n};' >> ./types/podlet.d.ts"
```

Comment:
Good idea to append `podiumSend` after `tsc` ❤️ For readability I'd love to have this in a separate script though.